default/scripting/fields/FLD_NEBULA_1.focs.txt

```diff
@@ -51,9 +51,12 @@ FieldType
             ]
             stackinggroup = "NEBULA_SHIP_EFFECTS"
             effects = [
-                SetSpeed value = max(5, Value - 20)
-                SetStealth value = Value + 20
-                SetDetection value = Value - 20
+                //SetSpeed value = max(5, Value - (NamedReal name = "FLD_NEBULA_SPEED_MALUS" value = 20))
+                //SetStealth value = Value + NamedReal name = "FLD_NEBULA_STEALTH_BONUS" value = 20
+                //SetDetection value = Value - NamedReal name = "FLD_NEBULA_DETECTION_MALUS" value = 20
```

Comment:
```suggestion

```

Reply:
Ouch... I forgot to remove the commented code.